### PR TITLE
Add FCHS

### PR DESCRIPTION
**Pull request by fchsLemke**
Please let me know if you need any additional information.

I am teaching a capstone course next year where we will be using C# to build an API from the ground up and React Native to build a mobile front-end. I've used Visual Studio with Jetbrains plugins as well as Rider at my industry jobs before transitioning into teaching, and I want my capstone students have access to a full-fledged editor before graduating.

Thank you so much for your time.

### CODE REVIEW

File: lib/domains/org/ftcsc.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Franklin Central High School
```
Comment (fchsLemke) [inline lib/domains/org/ftcsc.txt:1]: Staff emails

File: lib/domains/org/ftstudent.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Franklin Central High School
```
Comment (fchsLemke) [inline lib/domains/org/ftstudent.txt:1]: Student emails

**Comment by philipto** [comment]
@fchsLemke First stage of verification passed, official domain for **Franklin Central High School** is [ftcsc.org](https://ftcsc.org) as far as I verified. This is just an informational note. Pull request is still in the review. I appreciate your patience while waiting for the final decision.

**Comment by philipto** [comment]
@fchsLemke Thank you for your patience while waiting for a review and processing your request. Pull request merged.
**Please start requesting the licenses in about an hour, to let the changes to propagate through our system.**

**Pull request merged**